content/docs/Configuration/examples.md

```diff
@@ -73,6 +73,16 @@ create-archive:
 ```
 {{< /details >}}
 
+{{< details title="Creating archive with git submodules" >}}
+```yaml
+create-archive:
+  - git submodule update --init
+  # Create an archive file with the ‹pkg› directory prefix
+  - bash -c 'git ls-files --recurse-submodules | tar --transform "s|^|pkg/|" -caf .packit/pkg.tar.gz -T-'
+  - echo '.packit/pkg.tar.gz'
```

Comment:
I think you need to remove `.packit/` in the 2 commands above too. In my case, there is `.packit` directory` in the repository. But there is not `.packit` directory generally in a repository.